src/test/java/seedu/address/testutil/AppointmentBookBuilder.java

```diff
@@ -5,8 +5,6 @@
 
 /**
  * A utility class to help with building Appointmentbook objects.
- * Example usage: <br>
```

Comment:
why is the example usage removed?

Reply:
`withAppointment` takes in an actual `Appointment` object. So if an example is given, it will be quite verbose.
E.g. `AppointmentBook ab = new AppointmentBookBuilder().withAppointment(new Appointment(new Name("Meet Alex"), new Remark("remark"), new Date(LocalDate.parse("2021-01-01")), new Time(LocalTime.parse("20:00:00"))).build();`
So I don't think it is necessary to put. Let me know if yall think we should include it.

Reply:
that makes sense